File: Leetcode/9-Palindrome-Number/PalindromeNumber.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    var s = x.toString()
    var i = 0
    var j = s.length-1
    while (i<j) {
        if (s.charAt(i)!=s.charAt(j)) {
            return false
        }
        i++
        j--
    }
    return true
};

File: Leetcode/9-Palindrome-Number/PalindromeNumber.go
import (
	"strconv"
)

func isPalindrome(x int) bool {
	s := strconv.Itoa(x)
	i := 0
	j := len(s) - 1
	for i < j {
		if s[i] != s[j] {
			return false
		}
		i++
		j--
	}
	return true
}
